### ISSUE TITLE
GH Pages deploy is not running

### ISSUE THREAD

**Issue by hernangonzalez** [rust-lang-ar/rust-lang-ar.github.io#32]

**Comment by LeoBorai** [comment]
Looks like PRs are being merged against `development` instead of `main`.

pushing development commits to main should be enough.

**Comment by LeoBorai** [comment]
I suggest tagging and deploy tags instead perhaps?

**Comment by hernangonzalez** [comment]
I agree, trunk should be enough to manage this build imho. For now I will create a PR to main and close this issue. Thank you.
